memcnn/models/tests/test_split_dim.py

```diff
@@ -0,0 +1,32 @@
+
+import pytest
+import torch
+
+from memcnn import AdditiveCoupling, AffineAdapterNaive, AffineCoupling, InvertibleModuleWrapper
+
+
+class Check(torch.nn.Module):
+    def __init__(self, dim, target_size):
+        super(Check, self).__init__()
+        self.dim = dim
+        self.target_size = target_size
+
+    def forward(self, fn_input):
+        assert fn_input.size(self.dim) == self.target_size
+        return fn_input
+
+
+@pytest.mark.parametrize('dimension', [None, 0, 1, 2])
+@pytest.mark.parametrize('coupling', [AdditiveCoupling, AffineCoupling])
+def test_split_dim(dimension, coupling):
+    input_size = (4, 8, 12)
+    dim = 1 if dimension is None else dimension
+    module = Check(dim, input_size[dim] // 2)
+    coupling_args = dict(adapter=AffineAdapterNaive) if coupling.__name__ == 'AffineCoupling' else dict()
+    if dimension is not None:
+        coupling_args["split_dim"] = dimension
+    model = coupling(module, **coupling_args)
+    model = InvertibleModuleWrapper(model)
```

Comment:
The CI is failing on the last assert because the InvertibleModuleWrapper is being used and it puts the inp.shape to [0] in PyTorch version 0.4. It is better to remove it since it is not essential to the part being tested and will avoid this error.

Reply:
You're right, the invertible module wrapper isn't necessary.